semigroups.cabal

```diff
@@ -128,6 +136,9 @@ library
     if flag(bytestring)
       build-depends: bytestring >= 0.9 && < 1
```

Comment:
used here to split this into `bytestring >= 0.9 && <0.10, bytestring-builder >= 0.10.4 && < 1` and `bytestring >=0.10` parts. Like in http://hackage.haskell.org/package/time-locale-compat-0.1.1.2/time-locale-compat.cabal